pytests/tests/test_install.py

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (C) 2019 VMware, Inc. All Rights Reserved.
```

Comment:
2020

Reply:
these commits were authored in dec 2019. The commit dates also reflect this. Do you still want me to put them to 2020?

Reply:
Done.